proposals/nnnn-video-streaming-state.md

```diff
@@ -0,0 +1,165 @@
+
+#### Mobile API
+```xml
+<function name="OnHMIStatus" functionID="OnHMIStatusID" messagetype="notification">
+    <param name="hmiLevel" type="HMILevel" mandatory="true">
+        <description>See HMILevel</description>
+    </param>
+    
+    <param name="audioStreamingState" type="AudioStreamingState" mandatory="true">
+        <description>See AudioStreamingState</description>
+    </param>
+    
+    <param name="systemContext" type="SystemContext" mandatory="true">
+        <description>See SystemContext</description>
+    </param>
+    
+    <!-- new additions-->
+    <param name="videoStreamingState" type="VideoStreamingState" mandatory="true">
+        <description>See VideoStreamingState. 
+        If it is NOT_STREAMABLE, the app must stop streaming video to SDL (stop service).
+        </description>
+    </param>
+</function>
+
+<enum name="VideoStreamingState">
+    <description>Enumeration that describes possible states of video streaming. </description>
+    <element name="STREAMABLE" />
+    <element name="NOT_STREAMABLE" />
+</enum>
+```
+
+#### HMI PI
+This proposal also add a new enum value `PROJECTION` to data type `EventTypes`.
+HMI shall send a `OnEventChanged` notification with `EventTypes`=`PROJECTION` to SDL when HMI brings a projection app to foreground (`isActive`=`true`) or HMI switches from the foreground projection app (`isActive`=`false`).
+
+
+```xml
+<enum name="EventTypes">
+  <description>Reflects the current active event</description>
+  <element name="AUDIO_SOURCE">
+    <description>Navigated to audio(radio, etc)</description>
+  </element>
+  <element name="EMBEDDED_NAVI">
+    <description>Navigated to navigation screen</description>
+  </element>
+  <element name="PHONE_CALL">
+    <description>Phone call is active</description>
+  </element>
+  <element name="EMERGENCY_EVENT">
+    <description>Active emergency event, active parking event</description>
+  </element>
+  <element name="DEACTIVATE_HMI">
+    <description> GAL/DIO is active </description>
+  </element>
+  
+  <!-- new additions-->
+  <element name="PROJECTION">
+    <description>Projection app is shown in HMI</description>
+  </element>
+</enum>
+```
+
+#### Compatibility
+- New apps on old HUs: Old HUs do not support new projection type app. Since old HUs does not send the new videoStreamingState parameter, on mobile proxy, we can give default value as *streamable* to  the parameter and make sure app does not do streaming in `BACKGROUND` or `NONE`  to make new navigation apps work with old HUs.
```

Comment:
Please update the second sentence to "Since old HUs do not send..."

Reply:
updated